spec/support/helpers/model_spec_helpers.rb

```diff
@@ -0,0 +1,88 @@
+module ModelSpecHelpers
+  shared_context "question set" do
+    let!(:loan_type_set) { create(:option_set, model_type: ::Loan.name, model_attribute: 'loan_type') }
+    let!(:lt1) { create(:option, option_set: loan_type_set, value: 'lt1', label_translations: {en: 'Loan Type One'}) }
+    let!(:lt2) { create(:option, option_set: loan_type_set, value: 'lt2', label_translations: {en: 'Loan Type Two'}) }
+    let!(:loan1) { create(:loan, loan_type_value: lt1.value)}
+    let!(:loan2) { create(:loan, loan_type_value: lt2.value)}
+    let!(:qset) { create(:loan_question_set, internal_name: 'loan_criteria') }
+    let!(:root) { qset.root_group }
+    let(:rset) { build(:loan_response_set, loan: loan1) }
+  end
+
+  shared_context "full question set and responses" do
```

Comment:
Funny how we both just leveled up factories for these two hierarchy things (timeline, questions).

I ended up doing it a bit differently so that the fake timeline is accessible outside the spec world via FactoryGirl. We can compare notes later. It's pretty easy to move from one to the other.